Tools/machines/tuolumne-llnl/install_mi300a_dependencies.sh

```diff
@@ -143,6 +171,43 @@ cmake \
     --parallel ${build_procs}
 rm -rf ${build_dir}/lapackpp-tuolumne-mi300a-build
 
+# PETSC
+if [ -d ${SRC_DIR}/petsc ]
+then
+  cd ${SRC_DIR}/petsc
+  git fetch --prune
+  git checkout v3.24.0
+  cd -
+else
+  git clone -b v3.24.0 https://gitlab.com/petsc/petsc.git ${SRC_DIR}/petsc
+fi
+cd petsc
```

Comment:
Forgot the
```suggestion
cd ${SRC_DIR}/petsc
```
Fixed via #6410